drivers/bridge/bridge.go

```diff
@@ -1217,7 +1220,11 @@ func (d *driver) Join(nid, eid string, sboxKey string, jinfo driverapi.JoinInfo,
 	}
 
 	iNames := jinfo.InterfaceName()
-	err = iNames.SetNames(endpoint.srcName, containerVethPrefix)
```

Comment:
For readability, I would rename the constant to `defaultContainerVethPrefix` and here have

```
containerVethPrefix := defaultContainerVethPrefix
if network.config.ContainerIfacePrefix != "" {
    containerVethPrefix = network.config.ContainerInterfacePrefix
}
err = iNames.SetNames(endpoint.srcName, containerVethPrefix)
```